.travis.yml

```diff
@@ -4,7 +4,6 @@ python:
   - 3.3
   - 3.4
   - 3.5
-  - 3.6
```

Comment:
Good point!  Maybe `3.6-dev = py36` would be more stable, as long as travis has a `3.6-dev`.  I suppose when 3.6 final is out the `-dev` will also disappear and travis config will require changes in any case.

Anyway, thank you for teaching me that you *can* test Python 3.6 prereleases with tox-travis!